src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
I see that you did it in all files. this work cannot be approved until you remove that line of code